GDPR, including the Client ID

morsssss opened the pull request:
@bpaduch , can you help me see whether there's a build issue here?

This is just the site changes, not the ClientID stuff.  Much had changed since I did this, so I merged in the latest from master... there's still one change that's not mine that you may not want (the one toward the end of head.html)

bpaduch commented:
@morsssss - experiencing no build errors/issues.  Make sure you're using the latest [yarn, grow, etc](https://github.com/ampproject/docs#install).  Also, when you fix the other issues, deploy to "custom" so can be reviewed by Rudi and Eric.

morsssss commented:
@bpaduch, please check this out again!

Note though that this you can't dismiss the dialog box in an Incognito window.  This might be a bug report for AMP...

morsssss commented:
Actually, I tried this again, and now it works in Incognito. Strange. Maybe it was an issue with the device simulator...

bpaduch commented:
@pbakaus - please review.  I deployed it for Ben to the Custom site at: https://ampproject-custom.firebaseapp.com/

bpaduch commented:
Screenshots on local:

Desktop
![localhost_8080_](https://user-images.githubusercontent.com/1012254/43461887-9ac723d6-94a2-11e8-80fb-8aed590d020b.png)

Mobile
![localhost_8080_ pixel 2](https://user-images.githubusercontent.com/1012254/43461889-9ad503d4-94a2-11e8-9428-d4fc9630328c.png)

pbakaus commented on views/partials/head.html:
```diff
@@ -80,7 +82,7 @@
     {% elif doc.view == '/views/list_page.html' or doc.url.path.endswith('/latest/') %}
       {% include "/assets/css/list.min.css" %}
-    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.find('/roadmap/') != -1 %}
+    {% elif doc.view == '/views/blog_detail.html' or doc.url.path.endswith('/roadmap/') %}
```
This one needs to be reverted, as the `find` syntax is needed to not break the roadmap filter pages.

morsssss replied:
Thanks for catching that. It's an artifact of my attempt to merge my fork to reflect changes on master :-/

morsssss commented:
Ok.  @bpaduch or @pbakaus - please take a look! I reverted that unwanted change, and I also added the `<meta>` tag that's supposed to make Client ID work in GA.

pbakaus commented:
LGTM, thanks!
